modules/file-system/pre-existing-network-storage/scripts/mount-daos.sh

```diff
@@ -69,39 +80,33 @@ sed -i "s/#.*user_allow_other/user_allow_other/g" $fuse_config
 # make sure limit of open files is high enough for dfuse (1M of open files)
 ulimit -n 1048576
 
-for i in {1..10}; do
-	# To parse mount_options as --disable-wb-cache --eq-count=8.
-	# shellcheck disable=SC2086
-	dfuse -m "$local_mount" --pool default-pool --container default-container --multi-user $mount_options && break
-
-	echo "dfuse failed, retrying in 1 seconds (attempt $i/10)..."
-	sleep 1
-done
-
-if ! mountpoint -q "$local_mount"; then
-	exit 1
-fi
-
 # Store the mounting logic in a variable
-mount_command='for i in {1..10}; do /bin/dfuse -m '$local_mount' --pool default-pool --container default-container --multi-user '$mount_options' --foreground && break; echo \"dfuse, failed, retrying in 1 second (attempt '$i'/10)\"; sleep 1; done'
+mount_command="if mountpoint -q '$local_mount'; then fusermount3 -u '$local_mount'; fi; for i in {1..10}; do /bin/dfuse -m '$local_mount' --pool default-pool --container default-container --multi-user $mount_options --foreground && break; sleep 1; done"
+
+# Construct the service name with the local_mount suffix
+service_name="mount_parallelstore_${local_mount//\//_}.service"
 
 # --- Begin: Add systemd service creation ---
-cat >/usr/lib/systemd/system/mount_parallelstore.service <<EOF
+cat >/usr/lib/systemd/system/"${service_name}" <<EOF
```

Comment:
Should this land in `/usr/lib/systemd/system` or `/etc/systemd/system`? See [table 1 here](https://www.freedesktop.org/software/systemd/man/latest/systemd.unit.html). I'd lean towards putting those into `/etc/systemd/system`

Reply:
I agree.